### DISCUSSION TITLE
How to make Multi axis in Mixed line and bar chart

### DISCUSSION THREAD

**Discussion by kimdonghyun1219** [cloudscape-design/components#651] [Q&A] [answered]
make chart for cloudscape chart
but i have a problem has arisen
yscale want to scale left and right
please how to make in chart do multi axis line and bar chart

**Comment by just-boris** [comment, marked as answer]
Multiple axis are not supported in our charts components.

It is deliberate decision, because our research found that multiple axes confuse readers.

If you have some counter examples, please share in comments

**Reply by mastershif** [reply]
Is this still the case or did it change since?

**Reply by connorlanigan** [reply]
There has been no change. Multiple axes are not supported in Cloudscape's chart components.

**Reply by mastershif** [reply]
Thanks @connorlanigan 🙏
